client/src/components/ConfirmationDialog/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+    Button,
+    AlertDialog,
+    AlertDialogOverlay,
+    AlertDialogContent,
+    AlertDialogHeader,
+    AlertDialogBody,
+    AlertDialogFooter,
+} from "@chakra-ui/core";
+
+const ConfirmationDialog = ({ isOpen, onClose, confirm, itemType, itemName, action, navigateTo }) => {
+    const cancelRef = React.useRef();
+
+    return (
+        <>
```

Comment:
Is this empty tag supposed to be here o.0 sp00ky

Reply:
Yes sir, it is a react fragment